server/controllers/awsEmail.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
                 to: to,
                 subject: `Your friend ${options.fromAddress} gave you jobCents!`,
                 // text: '',
-                html: htmlTemplate.inviteHtml(options.challengeTitle, options.description, options.fromAddress, formatDollars(options.rewardAmount), options.participationUrl, options.company)
+                html: htmlTemplate.inviteHtml(to, options.challengeTitle, options.description, options.fromAddress, formatDollars(options.rewardAmount), options.participationUrl, options.company)
```

Comment:
are there any other placing calling the inviteHtml function? I want to make sure the addition of the new parameter at the front of the function call doesn't cause other uses to fail elsewhere in the code

Reply:
inviteHtml appears to only be called in one place